packages/markdown/processor/compile/rdme-figure.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function CompileRdmeFigure() {
+  const Compiler = this.Compiler;
+  const visitors = Compiler.prototype.visitors;
+
+  function rdmeFigure(node) {
+    return ['<RdmeFigure>', this.block(node), '</RdmeFigure>'].join('\n\n');
```

Comment:
What's this used for?

Reply:
It's getting deleted. Early exploration!